recipes/h3-pyspark/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: h3-pyspark
+  version: {{ version }}
+
+source:
+  url: https://github.com/kevinschaich/h3-pyspark/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 64c39a66664676ce799dbfb5cbd49d9a9d76926e5495dcc8ea580fb03b4b46fb
+
+build:
+  noarch: python
+  number: 0
+  # skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - pyspark >=3.0.0
+    - h3-py >=3.7.0
+    - shapely >=1.7.0
+
+test:
+  requires:
+    - python >=3.6
+    - pyspark >=3.0.0
+    - h3-py >=3.7.0
+    - shapely >=1.7.0
```

Comment:
Please remove those requirements.

The goal of the test is to check that the package can be installed properly with all its needed dependencies (that you define in run requirements).
`requires` in test is only to install extra things that could be required to run tests.